tensorflow/lite/testing/generate_examples_lib.py

```diff
@@ -286,6 +286,24 @@ def create_tensor_data(dtype, shape, min_value=-100, max_value=100):
       dtype)
 
 
+def create_random_infinite_data(dtype, shape):
```

Comment:
maybe instead of adding this one, you can change the create_tensor_data to support infinity, nan, etc.

Reply:
@renjie-liu , sure i have modified the same function and added support for infinity

Regards
Amit